torch/_inductor/decomposition.py

```diff
@@ -885,11 +885,13 @@ def select_decomp_table() -> dict[Any, Callable[..., Any]]:
     """decomps can change based on config"""
     if config.fallback_random:
         return decompositions
-    if config.fallback_embedding_bag_byte_unpack is True:
+    if config.fallback_embedding_bag_byte_unpack:
         # remove q_embedding_bag_byte_unpack_decomp from decompositions
         for k in decompositions.keys():
             if k == torch.ops.quantized.embedding_bag_byte_unpack.default:
-                del decompositions[k]
+                decompositions.pop(
```

Comment:
I think you can get rid of the for-loop by using pop.

Reply:
you are right!